<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cave</class>
 <widget class="QWidget" name="Cave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cave</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTextBrowser" name="text_output"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="user_input"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="cave_layout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
